access/acceptance/secret_acl_test.go

```diff
@@ -132,3 +147,18 @@ func testSecretACLResource(scopeName, principal, permission string) string {
 		}
 		`, scopeName, principal, permission)
 }
+
+// testAccTokenResource returns an configuration for an Example Widget with the provided name
+func testSecretACLResourceWithDefaultPrincipal(scopeName, default_principal, principal, permission string) string {
+	return fmt.Sprintf(`
+		resource "databricks_secret_scope" "my_scope" {
+			name = "%s"
+            initial_manage_principal = "%s"
+		}
+		resource "databricks_secret_acl" "my_secret_acl" {
```

Comment:
so to test this, you should only create `databricks_secret_scope` and check what ACLs are returned through `NewSecretAclsAPI` in a custom resource check. In this integration test you're changing existing resource, which is not really testing the case of the bug, so i'd ask to create new `TestAcc...` for it.

Reply:
to rephrase previous comment - can you remove `databricks_secret_acl.my_secret_acl`, because this test should verify if there's no ACL for `users` when `initial_manage_principal` is not set.